src/createBrowserLikeFetch.js

```diff
@@ -62,9 +62,12 @@ function createBrowserLikeFetch({
               const value = decodeURIComponent(valueRaw);
               const cookieDomain = cookieOptions.domain;
               if (cookieDomain && `.${cookieDomain}`.endsWith(`.${hostname.split('.').slice(-2).join('.')}`)) {
+                // remove null values from cookieOptions
+                const filteredOptions = Object.fromEntries(Object.entries(cookieOptions)
```

Comment:
do `forEntries` and `entries` get polyfilled for IE?

Reply:
i believe it gets transpiled by babel, if not there are a large number of cases across the one app ecosystem that would need to be updated.

Reply:
I thought so... just very paranoid with ie11 at the moment😅

Reply:
@JAdshead @infoxicator `fromEntries` fails in Node v10 I know. I hope these get polyfilled but this will fail in Node v10 without them from experience.

Reply:
this is also for server not browser, this supports node >= 12